lib/tutorials/serving-files.md

```diff
@@ -96,7 +124,7 @@ server.route({
 
 ## Directory handler
 
-In addition to the `file` handler, there is also a `directory` handler that allows you to specify one route to serve multiple files. In order to use it, you must specify a path with a parameter. The name of the parameter does not matter, however. You can use the asterisk extension on the parameter to restrict file depth as well. The most basic usage of the directory handler looks like:
+In addition to the `file` handler, inert also adds a `directory` handler that allows you to specify one route to serve multiple files. In order to use it, you must specify a path with a parameter. The name of the parameter does not matter, however. You can use the asterisk extension on the parameter to restrict file depth as well. The most basic usage of the directory handler looks like:
```

Comment:
As this isn't a reference to the variable, I'm not sure it should be uppercase I. The lib isn't: https://github.com/hapijs/inert